src/utils/passcode.js

```diff
@@ -0,0 +1,31 @@
+const bcrypt = require('bcryptjs');
+
+/**
+ * @param {String} passcode - plaintext passcode to be encrypted
+ * @param {Number} [saltRounds] number of salting rounds for bcrypt
+ * @return {String} hashed passcode
+ */
+const encodePasscode = (passcode, saltRounds = 10) => {
+  if (!passcode) return '';
+  const salt = bcrypt.genSaltSync(saltRounds);
+  const passHash = bcrypt.hashSync(passcode, salt);
+  return passHash;
+}
+
+/**
+ * is the plaintextPasscode's hash same as the storedPasscodeHash?
```

Comment:
nit: I understand what you mean here, but for future comments, let's avoid asking questions and instead be declarative as to what the code is doing.

A potential revision might be `Checks if the submitted passcode matches the hashed passcode`